carmm/analyse/counterpoise_onepot.py

```diff
@@ -194,7 +197,8 @@ def calculate_energy_ghost_compatible(calc, atoms=None, properties=['energy'],
     from ase.calculators.calculator import Calculator
     import subprocess
     Calculator.calculate(calc, atoms, properties, system_changes)
-    calc.write_input(calc.atoms, properties, system_changes, ghosts=ghosts)
+    # Write inputfiles. Scaled positions does not with empty sites.
+    calc.write_input(calc.atoms, properties, system_changes, ghosts=ghosts, scaled=False)
```

Comment:
In ase 3.22, `scaled` can not be set in `fhi_calc.parameters`.
In ase/calculators/aims.py line 387,  `scaled` is default to `None`, and will be set to `True` if atoms is periodic in all three dimensions.
```
    def write_input(self, atoms, properties=None, system_changes=None,
                    ghosts=None, geo_constrain=None, scaled=None, velocities=None):
        FileIOCalculator.write_input(self, atoms, properties, system_changes)

        if geo_constrain is None:
            geo_constrain = "relax_geometry" in self.parameters

        if scaled is None:
            scaled = np.all(atoms.get_pbc())
```